angular/projects/spark-angular/src/lib/components/sprk-tabbed-navigation/sprk-tabbed-navigation.component.spec.ts

```diff
@@ -117,6 +150,16 @@ describe('SprkTabbedNavigationComponent', () => {
     expect(testPanel2.classList.contains('sprk-u-HideWhenJs')).toEqual(false);
   });
 
+  it('resizing should set the aria-orientation correctly when window is >= 736', () => {
+    component.ariaOrientation(800, testElement);
+    expect(testElement.getAttribute('aria-orientation')).toEqual('horizontal');
```

Comment:
How come we don't have to call fixture.detectChanges() here?

Reply:
We are calling a function directly and not depending on the component to change.
We're testing the output of a function.